CONTRIBUTING.md

```diff
@@ -296,6 +296,89 @@ OS and is built by Android NDK. This often introduces compatibility issues as
 Android (specifically Termux) is not a standard platform. Do not expect there
 are exist package recipes available out-of-box.
 
+## Commit guidelines
+
+A commit message should look something like:
+```
+<commitType>(<repo>/<package>): (The change made/Short description of the change)
+
+[An optional but **highly recommended** commit message describing the changes made in the commit]
+
+[Fixes https://github.com/termux/termux-packages/issues/<issue number>]
+[Closes https://github.com/termux/termux-packages/pull_requests/<pr number>]
+```
+
+- `<commitType>` describes the type of commit. Types of commits:
+  - `upgpkg`: One or more package(s) was/were updated.
+    Optional extended commit message may include new list of features in the new version, and detailed list of changes in build scripts and/or patches
+  - `dwnpkg`: One or more package(s) was/were downgraded due to build issues or potential bugs
+    commit message should include summary of why the package(s) was/were downgraded. If the summary can't completely describe the reason for downgrade, extended commit message should contain full reason for downgrade.
+  - `enhance`: Enable a feature in a package that was earlier not enabled.
+    Optional (but highly recommended) extended commit message may contain detailed summary of the enabled feature(s) and a basic use case
+  - `chore`: Any housekeeping change or a change which does not affect the user in any way.
+  - `rebuild`: Rebuild a package to link against newer version of shared library
+    Special cases:
+    - When mass rebuilding packages depending on a major package (e.g. openssl), consider using this format:
+      ```
+      rebuild(deps:main/openssl): link against OpenSSL 3.0
+      ```
+  - `scripts`: Any change affecting our build scripts
+
```

Comment:
Also add `disable(<repo>/<pkg name>): disable a package due to some issue`. Reason to disable should be mandatory.